### ISSUE TITLE
getPriorityFeeEstimate returns undefined when there is an error

### ISSUE THREAD

**Issue by llegoelkelo** [helius-labs/helius-sdk#133]
If the Helius API returns an error, the sdk fails silently and returns undefined which is incorrect according to the sdk function signature.

This happens because the API returns `200` but the response contains a property called `error`. Here is the error response:

You get the following error:
```
{
  "jsonrpc": "2.0",
  "error": {
    "code": -32602,
    "message": "Invalid params",
    "data": "unknown variant `Base58`, expected one of `binary`, `base64`, `base58`, `json`, `jsonParsed` at line 1 column 147"
  },
  "id": "helius-sdk"
}
```

To trigger an error, try the following:
```
helius.rpc.getPriorityFeeEstimate({
  transaction: 'your tx',
  options: {
    transactionEncoding: UiTransactionEncoding.Base58,
    includeAllPriorityFeeLevels: true,
  },
})
```

**Comment by 0xIchigo** [comment]
gm @llegoelkelo! Is this error still occurring since merging #134, or can we consider this issue closed?

**Comment by llegoelkelo** [comment]
Hey @0xIchigo I don't think this can be closed just yet after reviewing the #134. If an error occurs, the sdk will continue to return undefined.

**Comment by 0xIchigo** [comment]
gm @llegoelkelo! Let me know if #138 helps resolve this issue!
